gladier/tools/globus/compute.py

```diff
@@ -9,10 +9,30 @@
 ComputeFunctionType = t.Union[t.Callable[[t.Any], t.Any], str]
 
 
-class GlobusComputeState(ActionState):
+class ComputeState(ActionState):
+    """
+    The Globus Compute State allows for executing registered functions on a POSIX
+    based system. It requires a registered python function and a valid Globus Compute
+    endpoint to run on.
+
+    Example:
+
+    .. code-block:: python
+
+        def random_int(high_val: int) -> int:
+            from random import randint
+
+            return randint(0, high_val)
+
+
+        random_step = GlobusComputeState(
+            function_to_call=random_int, function_parameters={"high_val": 3}
+        )
+    """
+
     function_to_call: ComputeFunctionType
     action_url = "https://compute.actions.globus.org/fxap"
-    endpoint_id: str = "$.input.globus_compute_endpoint"
+    endpoint_id: str = "$.input.compute_endpoint"
```

Comment:
This is a change that would propagate out to users if they had already started using `globus_compute_endpoint`. That user group probably only consists of me, so that should be fine. :)

Reply:
I put out a PR a little while back announcing `compute_endpoint` with the new name, and all docs and examples have already been updated, so we're a bit late to change.